From 7d2e1aeb2d67b51540bc98c3336086977417bbd3 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 8 Aug 2017 14:20:36 +0800
Subject: [PATCH 2349/5242] MLK-16158-4 ARM64: dts: imx8mq: change dsi phy pll
 'max-data-rate' to 1.5GHz

commit  5b40f9e5369fc769c4eee507abbb169bcc7a12be from
https://source.codeaurora.org/external/imx/linux-imx.git

Accoring to the PHY spec, the DSI PHY PLL on imx8mq platform
can support up to 1.5GHz data rate. So change the 'max-data-rate'
to 1.5GHz on imx8mq.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index f9e9152..0c9f79f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -382,7 +382,7 @@
 		assigned-clock-rate = <594000000>, <266000000>, <80000000>;
 		phy-ref-clkfreq = <27000000>;
 		data-lanes-num = <4>;
-		max-data-rate  = <800000000>;
+		max-data-rate  = <1500000000>;
 		power-domains = <&power 0>;
 		status = "disabled";
 	};
-- 
1.7.9.5

